### ISSUE TITLE
Precompute Dataset Mean and Std

### ISSUE THREAD

**Issue by bohaohuang** [bohaohuang/mrs#27]
Mean and std should be precomputed for each dataset, see https://forums.fast.ai/t/image-normalization-in-pytorch/7534/7
We need a helper function for computing mean and std for any dataset or list of img files. Also, we need to store this mean and std stats so it can be easily read when defining dataloader. Which means we might not able to have it defined in .config files anymore...

**Comment by bohaohuang** [comment]
The next step is to make this meta data move with ds

**Comment by bohaohuang** [comment]
fixed by #49
